src/main/java/org/opensearch/security/dlic/rest/api/MigrateApiAction.java

```diff
@@ -27,7 +27,7 @@
 import org.opensearch.action.bulk.BulkResponse;
 import org.opensearch.action.index.IndexRequest;
 import org.opensearch.action.support.WriteRequest.RefreshPolicy;
-import org.opensearch.action.support.master.AcknowledgedResponse;
+import org.opensearch.action.support.clustermanager.AcknowledgedResponse;
```

Comment:
This is not required for 2.1.0.

Reply:
The build was broken, it seems like this was accidentally introduced in https://github.com/opensearch-project/OpenSearch/pull/3662 into 2.1 - I think its harmless to keep at this point

Reply:
Including only the required changes in a PR would be better. When regressions happen, it'd be much easier to revert commits with clear PR/commit scope. The broken build was cause by a bug in OpenSearch, and it has been resolved.